src/QueryFactory.php

```diff
@@ -34,6 +34,18 @@ public function create(): Query
                 )
             );
             $query->setFilter($filter);
+        } elseif ($this->config->hasFilter()) {
+            $query->setFilter(Filter::applyQueryString($this->config->getFilter()));
+        }
+
+        // Select
+        if ($this->config->hasSelect()) {
+            $query->setSelectFields($this->config->getSelect());
+        }
+
+        // Limit
+        if ($this->config->hasLimit()) {
+            $query->setTop($this->config->getLimit());
```

Comment:
`Top` znamena maximalny pocet zaznamov na jednu page + API to nemusi respektovat, ... napr. dam `10000`, ale na stranu mi to da iba `100` ... preto sa to este v `Extractor.php` kontroluje -> ci sa ma pokracovat nacitanim dalsej page alebo nie.